### DISCUSSION TITLE
c++/c#

### DISCUSSION THREAD

**Discussion by ningxiajizong** [godotengine/godot-proposals#7739] [Scripting]
I hope to add C++, C# and other languages to Godot's built-in script editor, which means that you can directly use Godot's built-in editor to write C++/C# scripts

**Comment by wxlv97** [comment]
A common external code editor is preferable, reducing development costs and allowing the focus to be on core functionality, such as Vscode

**Reply by ryevdokimov** [reply]
Especially with plugins like this: [https://github.com/godotengine/godot-vscode-plugin](url)
